# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../at76c503a/eth-naming.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- at76c503a/at76c503.c.orig	2007-03-24 21:41:50.035377533 +0100
+++ at76c503a/at76c503.c	2007-03-24 21:43:01.699461437 +0100
@@ -218,9 +218,9 @@
 module_param(monitor_scan_max_time, int, 0400);
 MODULE_PARM_DESC(monitor_scan_max_time, "scan max channel time in MONITOR MODE (default: 600)");
 
-static char* netdev_name = "wlan%d";
+static char* netdev_name = "eth%d";
 module_param(netdev_name, charp, 0400);
-MODULE_PARM_DESC(netdev_name, "network device name (default is wlan%d)");
+MODULE_PARM_DESC(netdev_name, "network device name (default is eth%d)");
 
 
 #define DEF_RTS_THRESHOLD 1536
